write-version-to-file.sh

```diff
@@ -0,0 +1 @@
+echo "'use strict';" > $1 && echo "// This is a generated file. Do not edit." >> $1 && echo "export const version = \"$(git ls-files | grep -v "$1" | git hash-object --stdin)\";" >> $1
```

Comment:
What is this used for?

Reply:
It is used to write a hash of all tracked files to a file. This hash is used in a new "getVersion" command which is exposed by the VS Code extension. This command was to be used by the Xronos VS Code extension to check for compatibility. However, the command does not appear to be used at all currently. We could remove it.

Reply:
No need. Looks useful.